### DISCUSSION TITLE
Shark Attack - Netlify deployment - Command failed with exit code 1: npm run build

### DISCUSSION THREAD

**Discussion by lindseysatterfield** [nss-evening-cohort-14/evening-client-side#230] [Help Me!] [answered]
#### Have you searched the other help tickets?
Yes, with #132 there was a similar issue, but mine has a different error.

#### Describe your issue
I am trying to deploy my Shark Tank app on Netlify. All of my deployments have failed. I used `npm run build` and `build/` in my basic build settings on Netlify.

#### List any error messages that appear
No errors in my console, but here are the errors that caught my eye from my deployment log:
`Command failed with exit code 1: npm run build`
`Build failed due to a user error: Build script returned non-zero exit code: 2`

This is the error I'm most confused about on line 116 of my log: 
`6:52:07 PM: Cannot find file '../helpers/data/studentData' in './src/App'.`

 My  `'../helpers/data/studentData'` seems to be in the right place and everything is working fine on the app.

#### What have you googled?
https://answers.netlify.com/t/netlify-not-building-command-failed-with-exit-code-1-npm-run-build/16999
https://stackoverflow.com/questions/63314399/cant-deploy-react-project-error-is-command-failed-with-exit-code-1
https://answers.netlify.com/t/command-failed-with-exit-code-1-npm-run-build/20820

#### What have you tried? Tell us everything you have tried.
I tried many of the suggestions in the links above with the `CI= npm run build` in the build command and that didn't do anything. One other site suggested removing `package-lock.json`, `yarn.lock`, and then running `npm install` again. Tried that and nothing changed. 

#### Repo (not sure what file to show you for this error)
https://github.com/lindseysatterfield/shark-attack

#### File structure
![Screen Shot 2021-05-14 at 7 16 56 PM](https://user-images.githubusercontent.com/23327097/118342459-181f0d80-b4e9-11eb-8de1-2fd9951b3d98.png)

#### Deployment log
![deploy](https://user-images.githubusercontent.com/23327097/118342407-ddb57080-b4e8-11eb-8aa7-a7bc81f94838.png)

**Comment by TrinityChristiana** [comment, marked as answer]
`StudentData.js` form is actually capitalized. So change [this import](https://github.com/lindseysatterfield/shark-attack/blob/main/src/App/index.js#L7) to reflect that!

Netlify is really picky about capitalization.

**Reply by lindseysatterfield** [reply]
That was it! I wasn't getting any ES Lint errors, but you're right, Netlify didn't like it not being capitalized. Thanks!

Solution: I had to change `studentData.js` to `StudentData.js`
